src/ducks/connections/components/queue/queue.jsx

```diff
@@ -68,20 +72,28 @@ class Item extends Component {
         break
       case 'canceled':
         statusIcon = (
-          <Icon className="u-ml-half" icon="cross" color={palette['monza']} />
+          <Icon
+            className="u-ml-half"
+            icon={CrossIcon}
+            color={palette['monza']}
+          />
         )
         break
       case 'error':
       case 'conflict':
         statusIcon = (
-          <Icon className="u-ml-half" icon="warning" color={palette['monza']} />
+          <Icon
+            className="u-ml-half"
+            icon={WarningIcon}
+            color={palette['monza']}
+          />
         )
         break
       case 'done':
         statusIcon = (
           <Icon
             className="u-ml-half"
-            icon="check-circleless"
+            icon={CheckCirclelessIcon}
             color={palette['emerald']}
           />
         )
```

Comment:
Can we remove the use of the IconSprite?

Reply:
I haven't checked yet.